cmd/ops_agent_windows/run_windows.go

```diff
@@ -108,11 +114,12 @@ func (s *service) checkForStandaloneAgents(unified *confgenerator.UnifiedConfig)
 }
 
 func (s *service) generateConfigs() error {
-	data, err := ioutil.ReadFile(s.inFile)
+	confgenerator.MergeConfFiles(s.builtInConf, s.userConf, s.mergedConf, "windows")
+	data, err := ioutil.ReadFile(s.mergedConf)
 	if err != nil {
 		return err
 	}
-	uc, err := confgenerator.ParseUnifiedConfig(data, "windows")
+	uc, err := confgenerator.ParseUnifiedConfigAndValidate(data, "windows")
```

Comment:
I had to split the original method into two part (one for just urmarshal YAML, one for additional validation) in order to reuse the same code to read the user overrides config file, because the user overrides config does not map to a full UnifiedConfig struct.

Reply:
I don't follow - both should create a UnifiedConfig struct, which you can then merge together, and generate the output configs from.

Reply:
`ParseUnifiedConfig` https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/config.go#L42 does unmarshal and validation. For reaching the user overides config, it errors out because partial config fails the validation.